cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -1243,12 +1244,14 @@
       (high_part3 Reg (gen_select_reg (IntCC.Equal) shamt (zero_reg) (zero_reg) high_part2))
       (high Reg (alu_rrr (AluOPRRR.Or) high_part1 high_part3))
       ;;
-      (const64 Reg (load_u64_constant 64)))
+      (const64 Reg (load_u64_constant 64))
+      ;; right now we only rotate less than 64 bits.
+      (shamt_128 Reg (alu_andi (value_regs_get y 0) 127)))
     ;; right now we only rotate less than 64 bits.
     ;; if shamt is greater than 64 , we should switch low and high.
```

Comment:
This comment is outdated now, I think? This implementation does seem to support shifting by >= 64 bits, with the selects below.